packages/ozone/src/mod-service/lang.ts

```diff
@@ -0,0 +1,75 @@
+import { ModerationService } from '.'
+import { ModSubject } from './subject'
+import { ModerationSubjectStatusRow } from './types'
+
+export class ModerationLangService {
+  constructor(private moderationService: ModerationService) {}
+
+  async tagSubjectWithLang({
+    subject,
+    subjectStatus,
+    createdBy,
+  }: {
+    subject: ModSubject
+    createdBy: string
+    subjectStatus: ModerationSubjectStatusRow | null
+  }) {
+    if (
+      subjectStatus &&
+      !subjectStatus.tags?.find((tag) => tag.includes('lang:'))
+    ) {
+      try {
+        const recordLangs = await this.getRecordLang({
+          subject,
+        })
+        await this.moderationService.logEvent({
+          event: {
+            $type: 'com.atproto.admin.defs#modEventTag',
+            add: recordLangs
+              ? recordLangs.map((lang) => `lang:${lang}`)
```

Comment:
Small thing, just an idea.  But the language on a post could be highly specific, anything within BCP 47.  I wonder if for the purposes here it would make sense to use only the primary language tag.  E.g. `en-US` would be boiled down to `en`, not distinguishing between US english and british english `en-GB`.

Reply:
yes! I think this would be good to do

Reply:
It looks like the the first fragment of the hyphenated codes is the generalized code? That's what I'm storing now but not sure if we would want the full code to be there too?